### PR TITLE
Feat/add functionality to store current user

### PR DESCRIPTION
**Pull request by nickname038**

### CODE REVIEW

File: frontend/src/components/Routing/index.tsx [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
+    user,
+    isLoading,
+    error
+  }));
+  const { user, isLoading, error } = authStore;
+  console.log('auth', user, isLoading, error);
+
+  useEffect(() => {
+    localStorage.setItem('ACCESS_TOKEN', 'token');
+    authUser();
```
Comment (yevhenii-moroziuk) [inline frontend/src/components/Routing/index.tsx]: dispatch action only if token exists and use enum StorageKey enum

File: frontend/src/reducers/auth.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      ...state,
+      isLoading: false,
+      user: action.payload
+    };
+  },
+  [authAction.FAILURE](state, action: AuthFailureActionType) {
+    return {
+      ...state,
+      isLoading: false,
+      error: action.payload
```
Comment (yevhenii-moroziuk) [inline frontend/src/reducers/auth.ts]: no need to store the error, we will kick out the user anyway

File: frontend/src/actions/auth.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { createRoutine } from 'redux-saga-routines';
+import { User } from '../common/types/user';
+
+export enum authActions {
+  AUTH = 'AUTH'
```
Comment (yevhenii-moroziuk) [inline frontend/src/actions/auth.ts]: AUTH/FETCH_USER

**Review by yevhenii-moroziuk — CHANGES_REQUESTED** [review]

**Review by mtseluiko — CHANGES_REQUESTED** [review]
resolve conflicts

**Review by yevhenii-moroziuk — APPROVED** [review]

**Pull request merged**